contracts/test/recovery-test.ts

```diff
@@ -25,6 +29,7 @@ const signWalletAddress = async (
 
 describe("Recovery", async function () {
   const safetyDelaySeconds = 7 * 24 * 60 * 60;
+  const signatureExpiryWindow = 1200;
```

Comment:
Chose this arbitrarily. Is equal to 100 12 second blocks

Reply:
It looks to be used as an offset, so `signatureExpiryOffsetSeconds`? (I like putting units in where meaningful)

Reply:
Like that - agree the units make sense here. Will update to `signatureExpiryOffsetSeconds`